RELEASE-NOTES.md

```diff
@@ -4,6 +4,69 @@ section: overview
 releaseNoteTOC: true
 ---
 
+## 2020.02 release notes (2020-02-18)
+Packages released:
+- [@patternfly/react-catalog-view-extension: 1.4.5](https://www.npmjs.com/package/@patternfly/react-catalog-view-extension/v/1.4.5)
+- [@patternfly/react-charts: 5.3.3](https://www.npmjs.com/package/@patternfly/react-charts/v/5.3.3)
+- [@patternfly/react-core: 3.140.5](https://www.npmjs.com/package/@patternfly/react-core/v/3.140.5)
+- [@patternfly/react-inline-edit-extension: 2.17.5](https://www.npmjs.com/package/@patternfly/react-inline-edit-extension/v/2.17.5)
+- [@patternfly/react-styles: 3.7.2](https://www.npmjs.com/package/@patternfly/react-styles/v/3.7.2)
+- [@patternfly/react-table: 2.27.5](https://www.npmjs.com/package/@patternfly/react-table/v/2.27.5)
+- [@patternfly/react-tokens: 2.8.2](https://www.npmjs.com/package/@patternfly/react-tokens/v/2.8.2)
+- [@patternfly/react-topology: 2.14.5](https://www.npmjs.com/package/@patternfly/react-topology/v/2.14.5)
+- [@patternfly/react-virtualized-extension: 1.4.6](https://www.npmjs.com/package/@patternfly/react-virtualized-extension/v/1.4.6)
+- [@patternfly/react-icons: 3.15.0](https://www.npmjs.com/package/@patternfly/react-icons/v/3.15.0)
+
+### Components:
+- **Context selector:** Updated signature for context selector ([#3697](https://github.com/patternfly/patternfly-react/pull/3697))
+- **Data toolbar:** Updated html structure to match core ([#3701](https://github.com/patternfly/patternfly-react/pull/3710))
+- **Popover:** Added min-width override property ([#3601](https://github.com/patternfly/patternfly-react/pull/3601))
+- **Radio:** Added support for optional description ([#3621](https://github.com/patternfly/patternfly-react/pull/3621))
+- **Select:** Handled undefined default for checkbox select ([#3711](https://github.com/patternfly/patternfly-react/pull/3711))
+- **Simple list:** Added Simple list component ([#3645](https://github.com/patternfly/patternfly-react/pull/3645))
+- **Slider:** Added limit for lowest possible value of slider ([#3660](https://github.com/patternfly/patternfly-react/pull/3660))
```

Comment:
This should be moved to PF3 section under a "Slider". (See previous release notes)